pkg/client/session.go

```diff
@@ -80,7 +82,8 @@ func (c *immuClient) OpenSession(ctx context.Context, user []byte, pass []byte,
 	c.Options.DialOptions = dialOptions
 	c.SessionID = resp.GetSessionID()
 
-	c.HeartBeater = heartbeater.NewHeartBeater(c.SessionID, c.ServiceClient, c.Options.HeartBeatFrequency)
+	c.WithErrorHandler(c.keepAliveErrorHandler)
```

Comment:
isn't it overwriting the error handler that could be externally specified?

Reply:
the error handler can be already specified `func (c *immuClient) WithErrorHandler(handler heartbeater.ErrorHandler) *immuClient {`, but I think here is overwritten with `c.WithErrorHandler(c.keepAliveErrorHandler)`. Furthermore, `keepAliveErrorHandler` is panicking if there is an error and it shouldn't